python/cudf/cudf/bindings/groupby/groupby.pyx

```diff
@@ -83,7 +91,8 @@ def apply_groupby(keys, values, ops, method='hash', sort_results=True):
         else:
             c_ops.push_back(agg_names[ops[i]])
 
-    cdef hash_groupby.Options *options = new hash_groupby.Options()
+    cdef bool ignore_null_keys = dropna
+    cdef hash_groupby.Options *options = new hash_groupby.Options(dropna)
```

Comment:
Do you have to heap allocate the object? Can you not stack allocate?
```
cdef hash_groupby.Options options{dropna};
```

Reply:
Nope we can allocate this on the stack - will change

Reply:
Sorry - this is why I couldn't stack allocate:

```
cudf/bindings/groupby/groupby.pyx:95:30: C++ class must have a nullary constructor to be stack allocated
```